_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -1041,10 +1043,59 @@ private function _process_onboarding( $data ) {
 			}
 		}
 
+		// Add/update business address widget
+		if ( isset( $data['businessAddress'] ) ) {
+			//grab the first sidebar and proceed if it is present
+			$first_sidebar = Jetpack_Widgets::get_first_sidebar();
```

Comment:
This gives me 

> PHP Deprecated:  Non-static method Jetpack_Widgets::get_first_sidebar() should not be called statically in jetpack/_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php on line 1049

Since `get_first_sidebar()` doesn't seem to refer to the instance ( `$this`), I think we can actually make it `static`.